### PR TITLE
Finalized api routes

### PR DESCRIPTION
**Pull request by aniehuser**
@clundin55 this should help with writing the serializer

### CODE REVIEW

File: database/api/routes.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    uriVariable: [ 
+        {
+            id: Integer
+            port: Integer
+            name: String
+        }
+    ]
+    resource: [ 
+        {
+            id: Integer
```
Comment (clundin55) [inline database/api/routes.md:81]: Is it necessary to send the ID over JSON? Django can handle this for us and I can't think of a reason we would want to transfer IDs over

Comment (clundin55) [reply database/api/routes.md:81]: Note: This comment applies to all IDs found in the JSON

File: database/api/routes.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            name: String
+            uri: String
+            size: Integer
+            checksum: String
+            lastModified: String
+        }
+    ]
+    programBlock: [
+        {
+            id: Integer
```
Comment (clundin55) [inline database/api/routes.md:35]: is this id the id or the version id?

File: database/api/routes.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            inProgramBlock: Integer or NULL
+            name: String
+            qualifiedName: String
+        }
+    ]
+    data: [
+        {
+            id: Integer
```
Comment (clundin55) [inline database/api/routes.md:43]: Same comment can you specify id

File: database/api/routes.md (hunk 1 above)
Comment (aniehuser) [reply database/api/routes.md:81]: these arent the primary keys, they will be a part of a composite primary key with either `version_id` or `run_id`. check out the er diagram

File: database/api/routes.md (hunk 2 above)
Comment (aniehuser) [reply database/api/routes.md:35]: id. it is the id of the `data` table. the primary key of the `data` table on the server will be a composite key of the key from the database on yesworkflow, and its assciated version id

**Comment by clundin55** [comment]
LGTM

**Review by clundin55 — APPROVED** [review]

**Pull request merged**